config/attendize.php

```diff
@@ -54,14 +55,14 @@
     'question_radio_single'         => 6,
 
 
-    'default_timezone'           => 30, #Europe/Dublin
-    'default_currency'           => 2, #Euro
+    'default_timezone'           => 92, #Asia/Singapore
+    'default_currency'           => 15, #Philippine Peso
     'default_date_format'        => 'j M, Y',
     'default_date_picker_format' => 'd M, yyyy',
     'default_datetime_format'    => 'F j, Y, g:i a',
     'default_query_cache'        => 120, #Minutes
     'default_locale'             => 'en',
-    'default_payment_gateway'    => 1, #Stripe=1 Paypal=2 BitPay=3
+    'default_payment_gateway'    => 4, #Stripe=1 Paypal=2 BitPay=3 MIGS=4
```

Comment:
It's best to leave this default as Stripe as that is what the majority of people use. Same above for the currency.